x-pack/plugin/security/qa/security-trial/src/javaRestTest/java/org/elasticsearch/xpack/security/remoteaccess/RemoteAccessHeadersForCcsRestIT.java

```diff
@@ -393,9 +393,9 @@ private void expectActionsAndHeadersForCluster(
     }
 
     private void assertContainsRemoteClusterCredential(String clusterCredential, CapturedActionWithHeaders actual) {
-        assertThat(actual.headers(), hasKey(RemoteAccessHeaders.REMOTE_ACCESS_CLUSTER_CREDENTIAL_HEADER_KEY));
+        assertThat(actual.headers(), hasKey(RemoteAccessHeaders.REMOTE_CLUSTER_AUTHORIZATION_HEADER_KEY));
```

Comment:
"Authorization" aligns better with the setting name, and also the fact that the value itself is `ApiKey [encoded]` so essentially an authorization header value, so `REMOTE_CLUSTER_AUTHORIZATION_HEADER_KEY` feels more natural.